powerhome/playbook
# [NUXE-206] Unify use of error/error_message props across input kits

#### Screens
checkbox:
![Screen Shot 2020-10-27 at 12 10 11 PM](https://user-images.githubusercontent.com/64422843/97329435-67536880-184d-11eb-8fe5-72498b9ae388.png)
text input:
![Screen Shot 2020-10-26 at 4 25 20 PM](https://user-images.githubusercontent.com/64422843/97224940-484ecb00-17a8-11eb-8d01-d2946d286fa7.png)
date picker:
![Screen Shot 2020-10-26 at 4 25 54 PM](https://user-images.githubusercontent.com/64422843/97224942-484ecb00-17a8-11eb-9cba-cad875a3feae.png)
text area:
![Screen Shot 2020-10-27 at 9 15 22 AM](https://user-images.githubusercontent.com/64422843/97328872-d7adba00-184c-11eb-97bb-35cccca42c39.png)
radio:
![Screen Shot 2020-10-27 at 10 41 52 AM](https://user-images.githubusercontent.com/64422843/97328874-d7adba00-184c-11eb-8285-95e9acb7e914.png)
select:
![Screen Shot 2020-10-27 at 12 42 13 PM](https://user-images.githubusercontent.com/64422843/97334140-c23b8e80-1852-11eb-9587-0d28ea3bab1e.png)


#### Breaking Changes

YES. This is breaking because the error prop was previously a string, but in the version has changed to a boolean for consistency. The breaking kits are: `textarea`, `text_input`, `checkbox`, `date_picker`, and `select`. Radio kit is an addition.

#### Runway Ticket URL
https://nitro.powerhrg.com/runway/backlog_items/NUXE-206

#### How to test this

Check that these kits display an error intended and that the form kit generates propper real-time errors:
checkbox
text_input
date_picker
radio
textarea
select


#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
